File: 3633/3633.cpp
class Solution {
public:
    int earliestFinishTimeForLandAndWa(TreeNode* root) {
        if(!root) return 0;
        return 1+max(earliestFinishTimeForLandAndWa(root->left),earliestFinishTimeForLandAndWa(root->right));
    }
};

File: 3633/3633.cs
public class Solution {
    public int EarliestFinishTimeForLandAndWa(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(EarliestFinishTimeForLandAndWa(root.left), EarliestFinishTimeForLandAndWa(root.right));
    }
}